### ISSUE TITLE
[booth] consider in-one-go optimization for "new resource under a ticket": `pcs resource create NAME SPEC OPTS... --ticket TICKET`

### ISSUE THREAD

**Issue by jnpkrn** [ClusterLabs/pcs#227]
When knowinly creating a resource for multi-site clustering in one
cluster, it may be undesirable such a newly created resource would
come up on its own, even less so when running such commands in parallel
amongst the interconnected cluster (sites).

For instance, you don't want to (intermittently) confuse the network
with allowing the same virtual IP to come up in various clusters,
given this address shares the same network segment amongst the
clusters.

Workaround would to be create the resource with `--disabled` (not
detailed in the help message for `create`), but then, once you
have the ticket constraint defined, you would need to `enable`
that resource again.  It'd be more smart and smooth to allow to
create the resource with `--ticket TICKET`.  It would create
the respective `ticket` constraint along, meaning that, given
the circumstances, such resource would either stay down for
not having a ticket, or would be started, but at a single site
only (subject to mutual exclusion logic in booth).

IOW., in our motivation example, said confusion in the network
would be prevented while the configuration means at pcs side
would remain lightweight.

**Comment by tomjelinek** [comment]
Even though this may be a nice to have simplification, I don't see it getting done anytime soon due to its low severity compared to other tasks and an existence of a sufficient workaround (`pcs resource create --disabled`, `pcs constraint ticket add`, `pcs resource enable`).
